controllers/disruption_cron_controller.go

```diff
@@ -20,18 +22,31 @@ import (
 )
 
 type DisruptionCronReconciler struct {
-	Client  client.Client
-	Scheme  *runtime.Scheme
-	BaseLog *zap.SugaredLogger
-	log     *zap.SugaredLogger
+	Client      client.Client
+	Scheme      *runtime.Scheme
+	BaseLog     *zap.SugaredLogger
+	log         *zap.SugaredLogger
+	MetricsSink metrics.Sink
 }
 
 func (r *DisruptionCronReconciler) Reconcile(ctx context.Context, req ctrl.Request) (res ctrl.Result, err error) {
 	r.log = r.BaseLog.With("disruptionCronNamespace", req.Namespace, "disruptionCronName", req.Name)
 	r.log.Info("Reconciling DisruptionCron")
 
+	// reconcile metrics
+	r.handleMetricSinkError(r.MetricsSink.MetricReconcile([]string{"controller", r.MetricsSink.GetSinkName()}))
+
 	instance := &chaosv1beta1.DisruptionCron{}
 
+	defer func(tsStart time.Time) {
+		tags := []string{}
+		if instance.Name != "" {
+			tags = append(tags, "cronName:"+instance.Name, "namespace:"+instance.Namespace)
```

Comment:
IMO we should use the same tags in both logs and metrics: disruptionCronsomethingsomething

Reply:
Yeah that consistency sounds like it would be easier to correlate metrics and logs. I'll make the change.